api/src/main/java/io/strimzi/api/kafka/model/KafkaListenerExternal.java

```diff
@@ -19,37 +19,29 @@
 import com.fasterxml.jackson.annotation.JsonTypeInfo;
 
 /**
- * Configures the external listener which exposes Kafka outside of OpenShift
+ * Configures the external listener which exposes Kafka outside of Kubernetes / OpenShift
  */
 @JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")
 @JsonSubTypes({
         @JsonSubTypes.Type(name = KafkaListenerExternalRoute.TYPE_ROUTE, value = KafkaListenerExternalRoute.class),
+        @JsonSubTypes.Type(name = KafkaListenerExternalLoadBalancer.TYPE_LOADBALANCER, value = KafkaListenerExternalLoadBalancer.class),
+        @JsonSubTypes.Type(name = KafkaListenerExternalNodePort.TYPE_NODEPORT, value = KafkaListenerExternalNodePort.class),
 })
 @JsonInclude(JsonInclude.Include.NON_NULL)
 public abstract class KafkaListenerExternal implements Serializable {
     private static final long serialVersionUID = 1L;
 
-    //private KafkaListenerAuthentication serverAuthentication;
     private Map<String, Object> additionalProperties;
 
     @Description("Type of the external listener. " +
-            "Currently the only supported type is `route`. " +
-            "`route` type uses OpenShift Route for exposing Kafka to the outside.")
+            "Currently the supported types are `route`, `loadbalancer`, and `nodeport`. " +
```

Comment:
Just FYI you can use full asciidoc syntax here if you think it would be better formatted as bullets or a definition list.